packages/vite/src/rsc/rscBuildAnalyze.ts

```diff
@@ -24,16 +27,10 @@ export async function rscBuildAnalyze(viteConfigPath: string) {
   }
 
   await viteBuild({
-    configFile: viteConfigPath,
+    configFile: rwPaths.web.viteConfig as string,
```

Comment:
Is the typecast because `viteConfig` could be null or undefined or something? In that case I prefer an explicit check and a `throw`.